README.md

```diff
@@ -210,106 +185,91 @@ Further, the key manager interface **must** be passed as an argument to _all_ pu
 <!-- markdownlint-disable-next-line -->
 
-> [!NOTE]
-> ⚠️ = implemented but no test vectors present
+### Decentralized Identifiers
 
-### [`did:web`](https://w3c-ccg.github.io/did-method-web/)
+#### [DID Documents](https://www.w3.org/TR/did-core/) & [DID Resolution](https://w3c-ccg.github.io/did-resolution/)
 
-| Feature      | Typescript | Kotlin | Rust | Swift |
-| ------------ | ---------- | ------ | ---- | ----- |
-| `Resolution` | ❌          | ❌      | ⚠️    | ❌     |
+Independent of DID Methods, we support DID Documents according to [DID Core v1.0](https://www.w3.org/TR/2022/REC-did-core-20220719/) with the adjustments specified in [this corresponding document](spec/did.md).
 
-### [`did:jwk`](https://github.com/quartzjer/did-jwk/blob/main/spec.md)
+#### DID Methods
 
-| Feature      | Typescript | Kotlin | Rust | Swift |
-| ------------ | ---------- | ------ | ---- | ----- |
-| `Creation`   | ❌          | ❌      | ⚠️    | ❌     |
-| `Resolution` | ❌          | ❌      | ⚠️    | ❌     |
+| Method                                                 | Creation   | Resolution | Note |
+| ------------------------------------------------------ | ---------- | ---------- | ---- |
+| [`did:web`](https://w3c-ccg.github.io/did-method-web/) | ❌         | ✅         | -   |
+| [`did:jwk`](https://github.com/quartzjer/did-jwk/blob/main/spec.md) | ✅         | ✅  | - |
+| [`did:dht`](https://did-dht.com)                       | ✅         | ✅         | This is our default method. |
+| [`did:key`](https://w3c-ccg.github.io/did-method-key/) | ⚠️         | ⚠️         | Has been implemented in both Kotlin and Typescript, no plans for support in other languages. |
+| [`did:ion`](https://identity.foundation/sidetree/spec) | ⚠️         | ⚠️         | Support for `did:ion` has been deprecated. |
 
-### [`did:dht`](https://tbd54566975.github.io/did-dht-method/)
+### Verifiable Credentials
 
-| Feature      | Typescript | Kotlin | Rust | Swift |
-| ------------ | ---------- | ------ | ---- | ----- |
-| `Creation`   | ⚠️          | ⚠️      | ❌    | ❌   |
-| `Resolution` | ⚠️          | ⚠️      | ❌    | ❌   |
+#### [W3C Verifiable Credential Data Model 1.1](https://www.w3.org/TR/vc-data-model)
 
-### [`did:key`](https://w3c-ccg.github.io/did-method-key/)
+We support Verifiable Credentials according to the [Verifiable Credentials Data Model 1.1](https://www.w3.org/TR/vc-data-model) with the adjustments specified in [this corresponding document](spec/vc.md).
 
-| Feature      | Typescript | Kotlin | Rust | Swift |
-| ------------ | ---------- | ------ | ---- | ----- |
-| `Creation`   | ⚠️          | ⚠️      | ⚠️    | ❌   |
-| `Resolution` | ⚠️          | ⚠️      | ⚠️    | ❌   |
+| Feature                                                          |
+| ---------------------------------------------------------------- |
+| Data model                                                       |
+| Validation (data model, JSON Schema, status)                     |
```

Comment:
Do we want to really nail down specifically what validations / verifications here or elsewhere in the doc or is this too granular for this doc?

This is what we have in the [issue](https://github.com/TBD54566975/web5-js/issues/425) I created for this:

### Verification Processes

- **verify**: Runs all verification checks.

- **verifyIntegrity**: Performs a JWT integrity check to ensure proper formatting and inclusion of all necessary elements such as header, payload structure, and encoding checks.

- **verifySignature**: Cryptographically verifies the JWT's signature to confirm authenticity, involving issuer's DID resolution, public key retrieval, signature, and algorithm verification.

- **verifyVcDataModel**: Confirms the VC's structural compliance with relevant standards, verifying its validity as per the VC data model.

- **verifyExpiration**: Checks the VC expiration date to ensure validity.

- **verifyStatus**: Verifies the existence and validity of the status list credential, checking for revocation.

- **verifySchema**: Ensures VC adherence to a specified schema.

- **verifyTrustedIssuer**: Accepts options for a list of trusted issuer DIDs, marking critical importance with a warning symbol ⚠️.

Reply:
it's just all one giant verification system mannnnnnnn

in all seriousness, it's a good callout. my initial reaction is, are there any set of verification rules in any of the relevant specs for which we're like, "nah we're not doing that"? because maybe the assumption can be made we support all verification rulesets (which obv is easier said than done)

Reply:
I'm going to self-assign an issue to specify this. great callout

Reply:
https://github.com/TBD54566975/web5-spec/issues/123